source/index.html.md

```diff
@@ -1005,6 +1005,111 @@ supported.
   Also, <code>initiator</code> cannot be <code>merchant</code>.
 </p>
 
+##### Method: `token`
+
+Mastercard and Visa supply and require different sets of parameters for MDES and
+VTS transactions. Only one scheme can be used for a given authorization.
+
+<dl class="dl-vertical">
+  <dt>
+    token[visa]
+    <span class="type">dictionary</span>
+  </dt>
+  <dd>
+    Visa token data.
+    <div class="type">Optional. Cannot be present if <code>[mastercard]</code> is present.</div>
+  </dd>
+  <dt>
+    token[mastercard]
+    <span class="type">dictionary</span>
+  </dt>
+  <dd>
+    Mastercard token data.
+    <div class="type">Optional. Cannot be present if <code>[visa]</code> is present.</div>
+  </dd>
+</dl>
+
+###### Method: `token[mastercard]`
+
+<dl class="dl-vertical">
+  <dt>token[mastercard][tan]
+    <span class="type">[0-9]{12,19}</span>
+  </dt>
+  <dd>
+    Token Account Number to charge.
+    Found in <code>encryptedPayload.encryptedData.accountNumber</code> in the MDES response.
+  </dd>
+  <dt>token[mastercard][expiry]
+    <span class="type">[0-9]{6}</span>
+  </dt>
+  <dd>
+    Expiry.
+    Found in <code>encryptedPayload.encryptedData.applicationExpiryDate</code> in the MDES response.
+  </dd>
+  <dt>token[mastercard][cryptogram]
+    <span class="type">[:hex:]</span>
+  </dt>
+  <dd>
+    Token cryptogram.
+    Found in <code>encryptedPayload.encryptedData.de48se43Data</code> in the MDES response.
+  </dd>
+  <dt>
+    token[mastercard][3dsecure][v2]
+    <span class="type">dictionary</span>
+  </dt>
+  <dd>
+    See <a href="#authentication-3dsecure-v2">Authentication: [3dsecure][v2]</a>.
+    <div class="type">Optional</div>
+  </dd>
+</dl>
+
+##### Method: `token[visa]`
+
+<dl class="dl-vertical">
+  <dt>token[visa][tan]
+    <span class="type">[0-9]{16}</span>
+  </dt>
+  <dd>
+    Token Account Number (also known as Digital Account Number) to charge.
+  </dd>
+  <dt>token[visa][expire_month]
+    <span class="type">[0-9]{2}</span>
+  </dt>
+  <dd>
+    Expiry month.
+    Found in <code>tokenInfo.expirationDate.month</code> in the VTS response.
+  </dd>
+  <dt>token[visa][expire_year]
+    <span class="type">20[0-9]{2}</span>
+  </dt>
+  <dd>
+    Expiry year.
```

Comment:
```suggestion
    Expire year.
```

See my comment for line 1079.

Reply:
See the discussion in the other review comment thread. I'll resolve this as well.